.gitignore

```diff
@@ -106,3 +106,6 @@ target/
 
 # IPython
 .ipynb_checkpoints
+
+# .orig
+.py.orig
```

Comment:
hmm... actually, can we *not* hide the `.orig` files? Having them in my untracked files reminds me to clean up after merges. There's a reason that most `.gitignore`s don't, be default, include those.